### PR TITLE
feat: jira board collector/enricher

### PR DESCRIPTION
**Pull request by klesh**
feat: jira board collector/enricher
fix: docker requires shared mount for rabbitmq
feat: add issueResolvedAt/epicKey/status for jira enricherment
fix: fetcher should have default timeout

### CODE REVIEW

File: config/local.sample.js [hunk 1]
```diff
@@ -1,4 +1,7 @@
 module.exports = {
+  api: {
+    token: 'mytoken'
```
Comment (UltimateBeaver) [inline config/local.sample.js:3]: Could you please rename either `api` or `token` to indicate this token is for dev lake itself?

File: db/migrations/20210731152203-issue-created-updated.js [hunk 2]
```diff
@@ -2,16 +2,16 @@
...
+    })
+    await queryInterface.addColumn('jira_issues', 'issue_updated_at', {
+      type: Sequelize.DataTypes.DATE,
+    })
   },
 
   down: async (queryInterface, Sequelize) => {
-      await queryInterface.removeColumn('jira_issues', 'issue_created_at')
-      await queryInterface.removeColumn('jira_issues', 'issue_updated_at')
+    await queryInterface.removeColumn("jira_issues", "issue_created_at")
```
Comment (UltimateBeaver) [inline db/migrations/20210731152203-issue-created-updated.js:14]: Let's use single quotation mark for consistency.

File: db/migrations/20210731152203-issue-created-updated.js [hunk 3]
```diff
@@ -2,16 +2,16 @@
...
+    await queryInterface.removeColumn("jira_issues", "issue_updated_at")
```
Comment (UltimateBeaver) [inline db/migrations/20210731152203-issue-created-updated.js:15]: Same as above

File: db/migrations/20210803084926-create-jira-board.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('jira_boards', {
+      id: {
+        allowNull: false,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+        field: 'id'
+      },
+      projectId: {
```
Comment (UltimateBeaver) [inline db/migrations/20210803084926-create-jira-board.js:11]: A board is associated with one Jira project? Didn't we think that a board can have issues from different projects?

**Review by UltimateBeaver — COMMENTED** [review]
Looking good! Only some minor comments, could you please address them and rebase the branch? Then it'll be ready to merge, thanks! @klesh

File: db/migrations/20210803084926-create-jira-board.js (hunk 4 above)
Comment (klesh) [reply db/migrations/20210803084926-create-jira-board.js:11]: Board belongs to project, can have issues from different projects

**Pull request merged**